src/sfctl/params.py

```diff
@@ -209,3 +209,53 @@ def custom_arguments(self, _):  # pylint: disable=too-many-statements
         # expect the parameter command_input in the python method as --command in commandline.
         arg_context.argument('command_input',
                              CLIArgumentType(options_list='--command'))
+
+    with ArgumentsContext(self, 'mesh') as arg_context:
+        arg_context.argument('application_resource_name',
+                             options_list=('--app-name', '--application-name'),
+                             help="The name of the application", id_part='application_name')
```

Comment:
What does id_part do?

Reply:
This shouldn't have been there removing it